code/modules/lighting/lighting_system.dm

```diff
@@ -338,10 +338,7 @@
 		if(newalpha >= LIGHTING_DARKEST_VISIBLE_ALPHA)
 			newalpha = 255
 		if(lighting_object.alpha != newalpha)
-			if(instantly)
```

Comment:
the best way to find things calling redraw_lighting would be to comment out this proc, and look at the compile errors.